invokeai/frontend/web/src/features/nodes/components/sidePanel/WorkflowListMenu/WorkflowListItem.tsx

```diff
@@ -0,0 +1,155 @@
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleMouseOver = useCallback(() => {
+    setIsHovered(true);
+  }, []);
+
+  const handleMouseOut = useCallback(() => {
+    setIsHovered(false);
+  }, []);
+
+  const workflowId = useAppSelector(selectWorkflowId);
+  const downloadWorkflow = useDownloadWorkflow();
+
+  const { deleteWorkflow, deleteWorkflowResult } = useDeleteLibraryWorkflow(EMPTY_OBJECT);
+  const { getAndLoadWorkflow } = useGetAndLoadLibraryWorkflow({
+    onSuccess: () => $isWorkflowListMenuIsOpen.set(false),
+  });
+
+  const isActive = useMemo(() => {
+    return workflowId === workflow.workflow_id;
+  }, [workflowId, workflow.workflow_id]);
+
+  const handleClickLoad = useCallback(() => {
+    getAndLoadWorkflow(workflow.workflow_id);
+    $isWorkflowListMenuIsOpen.set(false);
+  }, [workflow.workflow_id, getAndLoadWorkflow]);
+
+  const handleClickEdit = useCallback(async () => {
+    await getAndLoadWorkflow(workflow.workflow_id);
+    dispatch(workflowModeChanged('edit'));
+    $isWorkflowListMenuIsOpen.set(false);
+  }, [workflow.workflow_id, dispatch, getAndLoadWorkflow]);
+
+  const handleDeleteWorklow = useCallback(() => {
+    deleteWorkflow(workflow.workflow_id);
+    onClose();
+  }, [workflow.workflow_id, deleteWorkflow, onClose]);
+
+  const handleClickDelete = useCallback(
+    (e: MouseEvent<HTMLButtonElement>) => {
+      e.stopPropagation();
+      onOpen();
+    },
+    [onOpen]
+  );
+
+  return (
+    <>
+      <Flex
+        gap={4}
+        onClick={handleClickLoad}
+        cursor="pointer"
+        _hover={{ backgroundColor: 'base.750' }}
+        py={3}
+        px={2}
+        borderRadius="base"
+        alignItems="flex-start"
+        w="full"
+        onMouseOver={handleMouseOver}
+        onMouseOut={handleMouseOut}
+      >
+        <Flex flexDir="column" w="full">
+          <Flex w="full" justifyContent="space-between" alignItems="flex-start">
+            <Flex alignItems="center" gap={2} w="full">
+              <Flex flexDir="column" gap={2} w="full">
+                <Flex alignItems="center" gap={1} justifyContent="space-between" w="full" minH={8}>
+                  <Flex alignItems="center" gap={3}>
+                    <Tooltip label={workflow.description}>
+                      <Text fontSize="md" noOfLines={2}>
+                        {workflow.name}
+                      </Text>
+                    </Tooltip>
+                    {isActive && (
+                      <Badge
+                        color="invokeBlue.400"
+                        borderColor="invokeBlue.700"
+                        borderWidth={1}
+                        bg="transparent"
+                        flexShrink={0}
+                      >
+                        {t('stylePresets.active')}
```

Comment:
maybe "selected" or "opened" or something like that is more appropriate than "active"